api_core/tests/unit/field_mask_test.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Google LLC
```

Comment:
Can you please just use a well-known type for this? The other tests use `http_pb2`, `date_pb2`, and `timeofday_pb2`

Reply:
Done!